### ISSUE TITLE
The sum of network consumption bill of pod is quite different from that of node level

### ISSUE THREAD

**Issue by wenxin-01** [kubesphere/kubesphere#3617]
**Environment**

http://192.168.18.71:30880

**How To Reproduce**
1. Go to 'toolbox'
2. Click on 'Metering and Billing'
3. Click on 'view consumption' in Cluster resource consumption
4. view node 'i-0ks3sxr7' in cluster 'host'
5. Count the consumption bill of pod and compare it with node ‘i-0ks3sxr7’

**Expected behavior**
The sum of network consumption bill of pod is equal to that of node level

**Actual behavior**
![image](https://user-images.githubusercontent.com/68640256/112793769-78243680-9098-11eb-89da-b85d768f6183.png)

[total_pod.xlsx](https://github.com/kubesphere/kubesphere/files/6219561/total_pod.xlsx)

/priority high
/kind bug
/assign @yunkunrao
/milestone 3.1.0

**Comment by yunkunrao** [comment]
Yeah, currently node level network consumption only cover north-south traffic and cross node pod network consumption. And pod level network consumption cover all scenarios which include not only above traffice but also include pod to pod comunication which resided on the same node. Currently it's expected.

**Comment by wenxin-01** [comment]
I hope it can be fixed in a later version

/priority low

**Comment by wenxin-01** [comment]
I hope it can be fixed in a later version

/priority low

**Comment by zryfish** [comment]
> I hope it can be fixed in a later version
> 
> /priority low

@wenxin-01 This issue has two priority labels now, is this still an urgent issue?

**Comment by wenxin-01** [comment]
> @wenxin-01 This issue has two priority labels now, is this still an urgent issue?
/remove-priority high

**Comment by xyz-li** [comment]
It's hard to distinguish pods's traffic on the same node from traffic between different nodes.
It's better to add a comment in document.

/cc @Patrick-LuoYu
